src/main/java/com/ironz/unsafe/UnsafeAndroid.java

```diff
@@ -239,6 +263,29 @@ public void unpark(Object obj) {
     }
 
     /**
+     * Stores an <code>boolean</code> field into the given object.
+     *
+     * @param obj      non-null; object containing the field
+     * @param offset   offset to the field within <code>obj</code>
+     * @param newValue the value to store
+     */
+    public void putBoolean(Object obj, long offset, boolean newValue) {
+        unsafe.putBoolean(obj, offset, newValue);
+    }
+
+    /**
+     * Stores an <code>boolean</code> field into the given object,
+     * using <code>volatile</code> semantics.
+     *
+     * @param obj      non-null; object containing the field
+     * @param offset   offset to the field within <code>obj</code>
+     * @param newValue the value to store
+     */
+    public void putBooleanVolatile(Object obj, long offset, boolean newValue) {
```

Comment:
Can't find this method in last sources https://android.googlesource.com/platform/libcore/+/android-o-preview-1/ojluni/src/main/java/sun/misc/Unsafe.java